portfolio/src/main/java/com/google/sps/servlets/RandomFactsAboutMeServlet.java

```diff
@@ -0,0 +1,97 @@
+package com.google.sps.servlets;
+
+import com.google.sps.data.FactAboutMe;
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+
+/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
+@WebServlet("/randomFactsAboutMe")
+public class RandomFactsAboutMeServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Create a ArrayList that will contains FactsAboutMe
+    ArrayList<FactAboutMe> factsAboutMeEN = new ArrayList<>();
+    ArrayList<FactAboutMe> factsAboutMeES = new ArrayList<>();
+    
+    // Add FactsAboutMeEn in English;
+    factsAboutMeEN.add( new FactAboutMe("images/logof1.png", "F1 logo","I'm a F1 fan since 2020"));
```

Comment:
I recommend abstracting this into two separate functions `getFactsAboutMeEN` and `getFactsAboutMeES` so that this `doGet` function is more readable.

e.g. 
```
ArrayList<FactAboutMe> factsAboutMeEN = getfactsAboutMeEN()
```